geoengine/datasets.py

```diff
@@ -404,3 +404,13 @@ class StoredDataset(NamedTuple):
 
     dataset_id: DatasetId
     upload_id: UploadId
+
```

Comment:
Could you update https://github.com/geo-engine/geoengine-python/blob/main/examples/raster_dataset.ipynb to see it "in action"?

Reply:
3ed742e81ebfd3804e8dab9fe75ee2f12ac2379d  be6ac17441fa48589196234a91a2add0f521f381